plugin.c

```diff
@@ -44,7 +44,11 @@ static DIR *open_nextdir(struct hts_path_itr *itr)
     DIR *dir;
 
     while (1) {
+#if defined(_WIN32) || defined(__MSYS__)
+        const char *colon = strchr(itr->pathdir, ';');
+#else
```

Comment:
Did you consider defining `PATH_SEPARATOR` as `':'` or `';'` as appropriate instead of sprinkling this guano everywhere and duplicating code? (Yes, you'll need `PATH_SEPARATOR_STRING` or other tricks as well.)